codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/BoringSSLCertificateCallback.java

```diff
@@ -37,7 +37,7 @@
 import java.util.Map;
 import java.util.Set;
 
-final class BoringSSLCertificateCallback {
+final public class BoringSSLCertificateCallback {
```

Comment:
why this change ?

Reply:
In order for a client or server to retrieve the allowed certificate type lists from the Certificate callback, the class needs to be made public rather then package protected.

Reply:
Reverted this change in favour of 2 new ssl context options